Content.Server/GameObjects/Components/GUI/StrippableComponent.cs

```diff
@@ -0,0 +1,371 @@
+                BreakOnTargetMove = true,
+                BreakOnUserMove = true,
+                NeedHand = true,
+            };
+
+            var result = await doAfterSystem.DoAfter(doAfterArgs);
+            if (result != DoAfterStatus.Finished) return;
+
+            userHands.Drop(item!.Owner, false);
+            inventory.Equip(slot, item!.Owner);
+
+            UpdateSubscribed();
+        }
+
+        /// <summary>
+        ///     Places item in user's active hand in one of the entity's hands.
+        /// </summary>
+        private async void PlaceActiveHandItemInHands(IEntity user, string hand)
+        {
+            var hands = Owner.GetComponent<HandsComponent>();
+            var userHands = user.GetComponent<HandsComponent>();
+            var item = userHands.GetActiveHand;
+
+            bool Check()
+            {
+                if (!ActionBlockerSystem.CanInteract(user))
+                    return false;
+
+                if (item == null)
+                {
+                    _notifyManager.PopupMessageCursor(user, Loc.GetString("You aren't holding anything!"));
+                    return false;
+                }
+
+                if (!userHands.CanDrop(userHands.ActiveHand!))
+                {
+                    _notifyManager.PopupMessageCursor(user, Loc.GetString("You can't drop that!"));
+                    return false;
+                }
+
+                if (!hands.HasHand(hand))
+                    return false;
+
+                if (hands.TryGetItem(hand, out var _))
+                {
+                    _notifyManager.PopupMessageCursor(user, Loc.GetString("{0:They} already {0:have} something there!", Owner));
+                    return false;
+                }
+
+                if (!hands.CanPutInHand(item, hand))
+                {
+                    _notifyManager.PopupMessageCursor(user, Loc.GetString("{0:They} cannot put that there!", Owner));
+                    return false;
+                }
+
+                return true;
+            }
+
+            var doAfterSystem = EntitySystem.Get<DoAfterSystem>();
+
+            var doAfterArgs = new DoAfterEventArgs(user, StripDelay, CancellationToken.None, Owner)
+            {
+                ExtraCheck = Check,
+                BreakOnStun = true,
+                BreakOnDamage = true,
+                BreakOnTargetMove = true,
+                BreakOnUserMove = true,
+                NeedHand = true,
+            };
+
+            var result = await doAfterSystem.DoAfter(doAfterArgs);
+            if (result != DoAfterStatus.Finished) return;
+
+            userHands.Drop(hand, false);
+            hands.PutInHand(item, hand, false);
+            UpdateSubscribed();
+        }
+
+        /// <summary>
+        ///     Takes an item from the inventory and places it in the user's active hand.
+        /// </summary>
+        private async void TakeItemFromInventory(IEntity user, EquipmentSlotDefines.Slots slot)
```

Comment:
Same here, you can do `
using static Content.Shared.GameObjects.Components.Inventory.EquipmentSlotDefines;`